cms/djangoapps/contentstore/views/access.py

```diff
@@ -3,6 +3,7 @@
 
 from common.djangoapps.student import auth
 from common.djangoapps.student.roles import CourseInstructorRole
+from openedx.core.djangoapps.course_roles.data import CourseRolesPermission
 
 
 def get_user_role(user, course_id):
```

Comment:
I will say, though it is not related to your work this is quite the misleading method and I would say it is pretty close to being an ticking-time-bomb security risk, right?